README.md

```diff
@@ -385,16 +385,27 @@ K9s uses aliases to navigate most K8s resources.
     refreshRate: 2
     # Number of retries once the connection to the api-server is lost. Default 15.
     maxConnRetry: 5
-    # Enable mouse support. Default false
-    enableMouse: true
-    # Set to true to hide K9s header. Default false
-    headless: false
-    # Set to true to hide K9s crumbs. Default false
-    crumbsless: false
     # Indicates whether modification commands like delete/kill/edit are disabled. Default is false
     readOnly: false
     # Toggles whether k9s should exit when CTRL-C is pressed. When set to true, you will need to exist k9s via the :quit command. Default is false.
     noExitOnCtrlC: false
+    #UI settings
```

Comment:
Looks like these are already in? Is your fork up to date?

Reply:
Yes I am on the latest commit, I am a bit confused by this note because the only "ui:" part the readme currently has is under skins

[You can also specify a default skin for all contexts in the root k9s config file as so:](https://github.com/derailed/k9s?tab=readme-ov-file#skins)

And I updated the section related to K9s config specifically which had the missing ui: parent ( for enableMouse headless etc ) , also it has an outdated logger.fullScreenLogs
[K9s configuration](https://github.com/derailed/k9s?tab=readme-ov-file#k9s-configuration)